src/compat/db_query.c

```diff
@@ -3572,6 +3572,91 @@ db_query_plan_dump_file (char *filename)
   return NO_ERROR;
 }
 
+/*
+ * db_query_get_plan_dump_file() -
+ * return : query dump file
+ */
+char *
+db_query_get_plan_dump_file ()
+{
+  return query_Plan_dump_filename;
+}
+
+/*
+ * db_query_is_plan_dump_opened() -
+ * return : is fp opened
+ */
+bool
+db_query_is_plan_dump_opened ()
+{
+  return query_Plan_dump_fp_open;
+}
+
+/*
+ * db_query_plan_dump_fp_open() -
+ * return : FILE *
+ */
+FILE *
+db_query_plan_dump_fp_open ()
+{
+  if (query_Plan_dump_fp_open)
```

Comment:
the status of plan dump fp can be checked by just checking query_Plan_dump_fp is NULL or not.

Reply:
At first, `query_Plan_dump_fp` is initialized to `NULL`. But In many places if `query_Plan_dump_fp` is `NULL`, set `query_Plan_dump_fp` to `stdout`. For readability, checking `query_Plan_dump_fp` is `NULL` or `stdout` seems not good for me.